lib/Makefile

```diff
@@ -4,11 +4,12 @@ endif
 
 include $(TOPDIR)/Makefile.global
 
-OBJECTS = $(addsuffix $(OBJEXT),lib.error lib.misc)
+OBJECTS = lib.error$(OBJEXT) lib.misc$(OBJEXT)
```

Comment:
Any reason for this change? I think their result is identical and I'd somewhat prefer the original.

Reply:
reverted